src/Mailer/Ownership/RecipientStrategy/SendToOwnersOfOrganisation.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership\RecipientStrategy;
+
+use CultuurNet\UDB3\Ownership\Repositories\OwnershipItem;
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\User\UserIdentityDetails;
+use CultuurNet\UDB3\User\UserIdentityResolver;
+use Psr\Log\LoggerInterface;
+
+class SendToOwnersOfOrganisation implements RecipientStrategy
+{
+    private UserIdentityResolver $identityResolver;
+    private DocumentRepository $organizerRepository;
+    private LoggerInterface $logger;
+
+    public function __construct(UserIdentityResolver $identityResolver, DocumentRepository $organizerRepository, LoggerInterface $logger)
+    {
+        $this->identityResolver = $identityResolver;
+        $this->organizerRepository = $organizerRepository;
+        $this->logger = $logger;
+    }
+
+    /**
+     * @return UserIdentityDetails[]
+     * @throws DocumentDoesNotExist
+     */
+    public function getRecipients(OwnershipItem $item): array
+    {
+        $organizer = $this->organizerRepository->fetch($item->getItemId())->getAssocBody();
+
+        //@todo loop over ALL owners of organisation
+        $ownerDetails = $this->identityResolver->getUserById($organizer['creator'] ?? '');
```

Comment:
I would make this a separate `SendToCreatorOfOrganisation` implementation of `RecipientStrategy`. If you make a `CombinedRecipientStrategy` you can make the combination you need. This is the same approach as the `AnyVoter`

Reply:
With this same approach you can make `Role RecipientStrategy` and `OwnershipRecipientStrategy`

Reply:
Implemented, really liked the idea.